src/Configuration/ConfigurationFactory.php

```diff
@@ -214,8 +213,6 @@ private static function retrieveCoverageBasePath(
         string $configDir,
         string $tmpDir
     ): string {
-        Assert::nullOrStringNotEmpty($existingCoveragePath);
```

Comment:
We do not care about its existence here: this responsibility is moved to the CoverageChecker